rtos/TARGET_CORTEX/mbed_rtx_handlers.c

```diff
@@ -43,9 +43,9 @@ __NO_RETURN uint32_t osRtxErrorNotify (uint32_t code, void *object_id)
     osThreadId_t tid = osThreadGetId();
 
     switch (code) {
-      case osRtxErrorStackUnderflow:
```

Comment:
Hmmm, @bulislaw, is this a part of the cmsis-rtos spec? Are we the only ones who depend on these names?